Yafc.Model/Model/ModuleFillerParameters.cs

```diff
@@ -1,10 +1,22 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace Yafc.Model {
     public interface IModuleFiller {
         void GetModulesInfo(RecipeParameters recipeParams, Recipe recipe, EntityCrafter entity, Goods? fuel, ref ModuleEffects effects, ref RecipeParameters.UsedModule used);
     }
 
+    [Serializable]
+    public record BeaconOverrideConfiguration(EntityBeacon beacon, int beaconCount, Module beaconModule) {
```

Comment:
What do you think about having a small summary that describes the purpose of this function? Or did you consider that the context is sufficient to the point that there's no need for summary?